libraries/ui-library/src/components/six-language-switcher/six-language-switcher.tsx

```diff
@@ -77,16 +77,25 @@ export class SixLanguageSwitcher {
         {this.languages.map((lang, index) => {
           const language = typeof lang === 'string' ? lang : lang.key;
           return (
-            <div onClick={this.handleLanguageSwitching(language, typeof lang === 'string' ? lang : lang.value)}>
+            <div
+              onClick={this.handleLanguageSwitching(language, typeof lang === 'string' ? lang : lang.value)}
+              onKeyDown={(e) => {
+                if (e.key === 'Enter' || e.key === ' ') {
+                  this.handleLanguageSwitching(language, typeof lang === 'string' ? lang : lang.value)();
+                }
+              }}
+            >
               <span
                 part="label"
+                tabindex="0"
```

Comment:
Looks good. Maybe we could also address the focus style at the same time. Something like this:

![image](https://github.com/six-group/six-webcomponents/assets/1475511/3bcbe65e-e50f-41eb-a232-de94ba9d1806)